spec/support/features/submit_stories_helpers.rb

```diff
@@ -0,0 +1,7 @@
+module Features
+  def fill_in_new_post_form(post)
+    fill_in 'Title', with: post.title
+    fill_in 'URL', with: post.url
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.